src/Microsoft.ML.TensorFlow/TensorflowTransform.cs

```diff
@@ -428,7 +428,7 @@ private protected override void SaveModel(ModelSaveContext ctx)
                 var buffer = Session.graph.ToGraphDef(status);
                 ctx.SaveBinaryStream("TFModel", w =>
                 {
-                    w.WriteByteArray(buffer.Data);
+                    w.WriteByteArray(buffer.MemoryBlock.ToArray());
```

Comment:
Same comment about `ToArray`

I'll stop commenting on them, and assume you will test the perf before and after these changes. And fix any perf degradations.